wip_pacta_data_qa.Rmd

```diff
@@ -0,0 +1,545 @@
+
+```{r}
+# read in if report has been run successfully, but data is not currently in the workspace
+total_portfolio <- paste0(project_location,"/30_Processed_Inputs/",project_name,"_total_portfolio.rda")
+bonds_company <- paste0(project_location,"/40_Results/Meta Investor/Bonds_results_company.rda")
+equity_company <- paste0(project_location,"/40_Results/Meta Investor/Equity_results_company.rda")
+
+
+result_data_qa <- readRDS(total_portfolio)
+bonds_results_company <- readRDS(bonds_company)
+equity_results_company <- readRDS(equity_company)
+
+```
+
+## Overview
+### variables, data types, etc.
+Brief overview of variable names and classes of the portfolio input file
+```{r}
+result_data_qa %>% glimpse()
+```
+
+### Total holdings and value
+Overall number of holdings and aggregated value in USD of the portfolio input, by asset type
+```{r}
+total_overview <- result_data_qa %>% 
+  filter(investor_name != "Meta Investor") %>% 
+  group_by(asset_type) %>% 
+  summarise(total_holdings = n(),
+            sum_value_usd = round(sum(value_usd, na.rm = T))) %>% 
+  ungroup()
+datatable(total_overview)
+```
+
+
+
+### Total holdings and value per investor
+Number of holdings and aggregated value in USD of the portfolio input - grouped by investor.
+"Meta Investor" is the total aggregation across investors.
+```{r}
+investors_overview <- result_data_qa %>% 
+  group_by(investor_name) %>% 
+  summarise(total_holdings = n(), sum_value_usd = round(sum(value_usd, na.rm = T))) %>% 
+  ungroup()
+datatable(investors_overview)
+```
+
+### Total holdings and value per investor and portfolio
+Number of holdings and aggregated value in USD of the portfolio input - grouped by investor and portfolio
+
+* **total_holdings** is the number of holdings per investor and portfolio
+* **sum_value_usd** is the aggregated value of the holdings in USD by investor and portfolio
+* **sum_market_value** .... see comment below
+* **pct_holdings** is the share of holdings of a specific PF, relative to the overall number of holdings in the analysis. It's the weight of the PF in pure terms of different holdings.
+* **value_weight** is the share of the value of a specific PF in USD, relative to the overall value across all PFs in the analysis. It shows how much value each of the PFs make up for
+
+**OPEN:**
+
+*sum_market_value* only makes sense if all holdings in a portfolio share the same currency. Otherwise, the overall value can look artificially inflated or deflated. Additionally grouping by currency would alleviate this, but the information on the PF level will not be readable anymore.
```

Comment:
Yeah I think this is the challenge. I think remove this "sum_market_value" as it could confuse I would suggest

Reply:
Yea, I'm just a bit unsure what to use for the missing currency table then, since this is where I need some info about a value in the original currency... maybe leave it up for discussion on Monday?